torch/_dynamo/variables/tensor.py

```diff
@@ -256,13 +260,31 @@ def unpack_var_sequence(self, tx, idxes=None):
             idxes = range(length)
         return [wrap_fx_proxy(tx, self.as_proxy()[i], **options) for i in idxes]
 
+    def _strict_mode_banned_ops(self):
+        return [
+            "stride",
```

Comment:
dtype should be here, though you did mention that it did cause a graph break in the model you were looking at. We should figure out if we want to include it or not

Reply:
> Left some comments and minor nits, the two points I want to make sure of is:
> 
> * safety: that we're disallowing all the methods/attrs we should be for a safe backwards trace

Yea, this is tricky, because we want to balance restrictiveness with perf because we are being overcautious. The attrs are not necessarily bad, but control flow on them is. I might make this a config driven list.

> * question about the testing plan: I remember the set_materialize_grads thing ended up exposing a bug where Dynamo would straight up yell instead of gracefully fallback to eager. It would be bad if we shipped this PR and then regressed existing user code that do have autograd.Function in them esp because autograd.Function is a very popular API. Do we have a good set of test cases for autograd.Function? (like, in torchbench?)

I don't know! More tests is better though, I am happy to add more tests. Fwiw - benchmark models have autograd.Function in them, and they pass, which is a good representation of reality. 

>   * Related: is it worth adding a dynamo config flag to turn off this feature? If config is True, then always graph break on autograd.Function. Or maybe that is just torch._dynamo.disallow_in_graph, or torch._dynamo.disable?

I can add a kill flag for this, on by default. 

> Otherwise, code LGTM

Thanks, responded to comments.

Reply:
Move to config.